packages/logs/src/boot/startLogs.ts

```diff
@@ -18,14 +18,14 @@ import type { LogsSessionManager } from '../domain/logsSessionManager'
 import { startLogsSessionManager, startLogsSessionManagerStub } from '../domain/logsSessionManager'
 import { startLoggerBatch } from '../transport/startLoggerBatch'
 import type { LogsConfiguration } from '../domain/configuration'
+import type { LogsEvent } from '../logsEvent.types'
 
-export function startLogs(configuration: LogsConfiguration, errorLogger: Logger) {
+export function startLogs(configuration: LogsConfiguration, logger: Logger) {
   const internalMonitoring = startInternalMonitoring(configuration)
 
   const errorObservable = new Observable<RawError>()
-
+  const consoleObservable = initConsoleObservable(['log', 'debug', 'info', 'warn', 'error'])
   if (configuration.forwardErrorsToLogs) {
-    trackConsoleError(errorObservable)
     trackRuntimeError(errorObservable)
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏careful about not loosing current behavior

Reply:
Fixed and I added the feature flag